charts/tfy-karpenter-config/templates/controlplane/karpenter-control-plane-ec2nodeclass.yaml

```diff
@@ -23,6 +23,8 @@ spec:
         {{ $key }}: {{ $val | quote }} 
         {{- end }}
   tags:
+    cpu: "true"
```

Comment:
we should rely on a tag, which can take both values. and then use filter on top, 
example: `compute_type: cpu|gpu`

Reply:
+1